app/helpers/self_service/self_service_helper.rb

```diff
@@ -45,9 +46,12 @@ def office_address_table_cell(office)
         .join(', ')
     end
 
-    def render_onboarding_message(page)
-      render partial: "self_service/onboarding/#{page}",
-             locals: { principal: current_user.principal }
+    def render_onboarding_message
+      return if current_user.principal.onboarded?
+
+      content_for :onboarding_message do
+        render 'shared/onboarding_message'
+      end
```

Comment:
Something seems a little indirect about this. Is there any significant difference in outcome if we ditch the `content_for` and render it directly?